tests/models/encoder_decoders/nlvr/nlvr_semantic_parser_test.py

```diff
@@ -48,3 +48,13 @@ def test_forward_with_epoch_num_changes_cost_weight(self):
         # The config file has ``wait_num_epochs`` set to 0, so the model starts decreasing the const
         # weight at epoch 0 itself.
         assert_almost_equal(cost_weights, [0.72, 0.648, 0.5832, 0.52488])
+
+
+class NlvrSemanticParserGroupedTest(ModelTestCase):
```

Comment:
You don't need another class for this.  See here: https://github.com/allenai/allennlp/blob/31ff177ccca06f044add05cda31f739b8ed7b3fc/tests/models/encoder_decoders/wikitables/wikitables_semantic_parser_test.py#L19-L24

Reply:
Done. Thanks!